numpy/ma/core.pyi

```diff
@@ -24,6 +24,7 @@ from numpy import (
     intp,
     ndarray,
 )
+from numpy._core.fromnumeric import _Array1D
```

Comment:
It's usually better to redefine relatively simple type-aliases that aren't intended to be used outside of the module where it's defined. Adding it to `numpy._typing` would also be an option.

Reply:
thanks for your review! sure, have redefined